database/postgres/queries.js

```diff
@@ -0,0 +1,78 @@
+const { generateData } = require('../data.js')
+const md5 = require('md5');
+// raw queries for seed script
+exports.dropTables = `
+drop table if exists products;
+drop table if exists descriptions;
+drop table if exists info;
+drop table if exists configurations;
+drop table if exists similarItems;
+`;
+
+exports.productTable = `
+  create table if not exists products(
+    id varchar(32) not null,
+    itemName text,
+    PRIMARY KEY(id)
+  )
+`;
+
+
+exports.descriptionsTable = `
+  create table if not exists descriptions(
+    id varchar(32) not null,
+    itemDescription json,
+    PRIMARY KEY(id)
+  )
+`;
+
+exports.infoTable = `
+  create table if not exists info(
+    id varchar(32) not null,
+    itemColor text,
+    brand text,
+    isPrimeFreeOneDay boolean,
+    isFreeDelivery boolean,
+    PRIMARY KEY(id)
+  )
+`
+
+exports.configurationsTable = `
+  create table if not exists configurations(
+    id varchar(32) not null,
+    configuration json,
+    PRIMARY KEY(id)
+  )
+`;
+
+exports.similarItemsTable = `
+  create table if not exists similarItems(
+    id varchar(32) not null,
+    similarItems json,
+    PRIMARY KEY(id)
+  )
+`;
+
+exports.pgRecord = (id) => {
+  const [record] = generateData(id, id);
+  const productId = md5(id);
```

Comment:
What is the benefit of you creating the ID (and specifically using md5 hash) instead of letting postgres add a uuid for the id in the schema?

Reply:
Using a consistent hash from 1-10M and using them as ids would allow everyone in the team to independantly create the exact same ProductIds no matter what database they were using. I saw that generating UUID v3 and v5 was possible on postgres but didnt test if it would be consistent across different databases.  No specific reason I was using md5 over v3 or v5.

Reply:
After testing a number of different refactors of the schema and insertion process I discovered that the number of hashes was nearly tripling the seed time and was right around 1 hour without them. WIll test other methods for hashing and uploading while maintaining the current schema and functionaility.